cross/tcl/PLIST

```diff
@@ -0,0 +1,257 @@
+rsc:bin/sqlite3_analyzer
```

Comment:
Reread the wiki please ;) If this file is part of the PLIST, then it should be referred to as `bin:`